Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3153[Validation] Avg Loss: 0.3151, Avg Dice Score: 0.8914 100% ||[Finetuning] Epoch: 2 Avg Loss: 0.3066[Validation] Avg Loss: 0.3164, Avg Dice Score: 0.8866100%[Finetuning] Epoch: 3 Avg Loss: 0.3014[Validation] Avg Loss: 0.3203, Avg Dice Score: 0.8779100%[Finetuning] Epoch: 4 Avg Loss: 0.2977[Validation] Avg Loss: 0.3286, Avg Dice Score: 0.8671100%[Finetuning] Epoch: 5 Avg Loss: 0.2955[Validation] Avg Loss: 0.3141, Avg Dice Score: 0.8883100%[Finetuning] Epoch: 6 Avg Loss: 0.2944[Validation] Avg Loss: 0.3142, Avg Dice Score: 0.8890100%[Finetuning] Epoch: 7 Avg Loss: 0.2920[Validation] Avg Loss: 0.3214, Avg Dice Score: 0.8773100%[Finetuning] Epoch: 8 Avg Loss: 0.2894[Validation] Avg Loss: 0.3209, Avg Dice Score: 0.8824100%[Finetuning] Epoch: 9 Avg Loss: 0.2908[Validation] Avg Loss: 0.3173, Avg Dice Score: 0.8839100%[Finetuning] Epoch: 10 Avg Loss: 0.2902[Validation] Avg Loss: 0.3152, Avg Dice Score: 0.8880/home/mikhelson/Meder143/143 [00:13<00:00, 10.66it/s]143/143 [00:13<00:00, 10.60it/s]143/143 [00:13<00:00, 10.68it/s]143/143 [00:13<00:00, 10.61it/s]143/143 [00:13<00:00, 10.57it/s]143/143 [00:13<00:00, 10.62it/s]143/143 [00:13<00:00, 10.55it/s]143/143 [00:13<00:00, 10.56it/s]143/143 [00:13<00:00, 10.58it/s]143/143 [00:13<00:00, 10.54it/s]ImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'TrainTest Results - Average Loss: 0.6161, Average Dice Score: 0.7228